modules/openapi-generator/src/main/resources/kotlin-client/infrastructure/RequestConfig.kt.mustache

```diff
@@ -8,10 +8,10 @@ package {{packageName}}.infrastructure
  * NOTE: Headers is a Map<String,String> because rfc2616 defines
  *       multi-valued headers as csv-only.
  */
-{{#nonPublicApi}}internal {{/nonPublicApi}}data class RequestConfig(
+{{#nonPublicApi}}internal {{/nonPublicApi}}data class RequestConfig<T>(
     val method: RequestMethod,
     val path: String,
     val headers: MutableMap<String, String> = mutableMapOf(),
     val query: MutableMap<String, List<String>> = mutableMapOf(),
-    val body: kotlin.Any? = null
+    val body: T? = null
```

Comment:
I wonder if this change has other hidden implications...

Reply:
Others than providing stronger (or previously missing) type information, I don't think so. I generated code for several combinations of `library` and `serializationLibrary` but didn't run all of them. However, previously not a single combination worked reliably.

Reply:
Just to be clearer, I like this change, I think it makes sense, I was just thinking if this could have some side effects that I didn't think of.